core/account-request-handler/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "itc-https-request-handler"
+name = "itc-account-request-handler"
```

Comment:
The reason I changed is: because I think here it is going to handle all account (twitter/discord/etc.) verification. Let's also see what @zhizming-zhong would say.

Reply:
It will be used to fetch data from other data providers (e.g. litentry graph) later, identity verification is one the usecases, so it's a generic https client implementation to me.